tests/test_translation_plugin.py

```diff
@@ -1,170 +1,90 @@
-##############################################################################
```

Comment:
Could you please put this banner back?

Reply:
This file was deleted, can you please resolve?

Reply:
Yes